contrib/endpoints/src/api_manager/service_control/proto.cc

```diff
@@ -993,6 +1039,28 @@ Status Proto::FillReportRequest(const ReportRequestInfo& info,
   return Status::OK;
 }
 
+// TODO:: jaebong::
+Status Proto::ConvertAllocateQuotaResponse(
+    const ::google::api::servicecontrol::v1::AllocateQuotaResponse& response,
+    const std::string& service_name, QuotaResponseInfo* quota_response_info) {
+
+  //response.operation_id()
+
+  if (response.allocate_errors().size() == 0) {
+    return Status::OK;
+  }
+
+  const QuotaError& error = response.allocate_errors().Get(0);
+  switch (error.code()) {
```

Comment:
do we need a switch here?

do we need QuotaResponseInfo?

I did not find places used.